tests/services/test_file_workflow_service.py

```
@@ -1177,4 +1179,343 @@ def test_record_file_in_db_legacy_path(monitored_path, db_session, tmp_path):
+        operation_type=OperationType.MOVE,
+    )
+    db_session.add(file_record)
+    db_session.commit()
+
+    # 2. Mock a remote listing where the file has been re-uploaded to Google Drive.
+    # The new file_id is "new-file-id", and the filename is "ffenc_99999.ffenc".
+    class MockSelfHealingBackend:
+        def backend_name(self) -> str:
+            return "gdrive"
+
+        def build_reference(self, location, path) -> str:
```

Ruff (0.15.15)

[warning] Unused method argument: `location`

(ARG002)